server/odc-service/src/main/java/com/oceanbase/odc/service/notification/helper/ChannelConfigValidator.java

```diff
@@ -90,10 +92,12 @@ private void validateWebhookChannelConfig(WebhookChannelConfig channelConfig) {
         Verify.verify(
                 channelConfig.getWebhook().startsWith("http://") || channelConfig.getWebhook().startsWith("https://"),
                 "Webhook should start with 'http://' or 'https://'");
-        Verify.verify(SSRFChecker.checkUrlInWhiteList(channelConfig.getWebhook(),
-                integrationConfigProperties.getUrlWhiteList()),
-                "The webhook is not in white list, please add it into system configuration with the key 'odc.integration.url-white-list'");
-
+        try {
```

Comment:
we may use both whitelist and blacklist, the mechanism enabled while value is not empty. and if whitelist is set, blacklist will not works.

Reply:
done. Only check blacklist when whitelist is empty.